az-core/src/main/java/azkaban/Constants.java

```diff
@@ -224,6 +224,21 @@
     public static final String QUEUEPROCESSING_ENABLED = "azkaban.queueprocessing.enabled";
 
     public static final String SESSION_TIME_TO_LIVE = "session.time.to.live";
+
+    // allowed max size of project dir in mb
+    public static final String PROJECT_DIR_MAX_SIZE = "azkaban.project.dir_max_size";
```

Comment:
Consider adding the unit in the key name? e.g.
dir_max_size_in_mb?

Do you foresee other keys in the project namespace? Project-dir can be interpreted as an individual project directory. 
Would it be better to name it 
azkaban.project_cache_max_size_in_mb?

Reply:
updated  in https://github.com/azkaban/azkaban/pull/1812